packages/appstore/src/components/currency-switcher-card/demo/demo-account-card.tsx

```diff
@@ -18,19 +18,11 @@ const DemoAccountCard = () => {
     const { accounts, loginid, resetVirtualBalance, default_currency } = client;
     const { selected_account_type } = traders_hub;
     const platform_demo_account = usePlatformDemoAccount();
-    const cfd_demo_accounts = useCFDDemoAccounts();
-    const cfd_demo_balance = useTotalAccountBalance(cfd_demo_accounts);
-    const cfd_demo_rate = useCurrencyExchangeRate(platform_demo_account?.currency || default_currency);
 
     const canResetBalance = () => {
         return loginid && accounts[loginid]?.balance !== 10000;
     };
 
-    const DemoAccountTotalBalance = React.useMemo(
-        () => (platform_demo_account?.balance || 0) + cfd_demo_balance.balance * cfd_demo_rate,
-        [cfd_demo_balance.balance, cfd_demo_rate, platform_demo_account?.balance]
-    );
```

Comment:
@shayan-deriv No need to calculate anymore? 🤔

Reply:
no. I comprared it with production. this should be fine